src/Service/Kinesis/src/Enum/EncryptionType.php

```diff
@@ -3,10 +3,10 @@
 namespace AsyncAws\Kinesis\Enum;
 
 /**
- * The encryption type used. This value is one of the following:.
+ * The server-side encryption type used on the stream. This parameter can be one of the following values:.
```

Comment:
php-cs-fixer incorrectly changed this doc. it should actually be

```
The server-side encryption type used on the stream.

This parameter can be one of the following values:

 - `NONE`: Do not encrypt the records in the stream.
 - `KMS`: Use server-side encryption on the records in the stream using a customer-managed AWS KMS key.

Reply:
note in particular the lack of a period after the colon

Reply:
> php-cs-fixer incorrectly changed this doc.

well, more that we were miss-using the short description area of the phpdoc. it is only meant to span either one line or one sentence.

Reply:
You're right, but it's not related to this PR. 
The code is generated, then cs-fixed, We can fix this later